application-xpoll-api/src/main/resources/META-INF/components.txt

```diff
@@ -1,3 +1,6 @@
 com.xwiki.xpoll.internal.DefaultXPollManager
 com.xwiki.xpoll.internal.rest.DefaultXPollResource
+com.xwiki.xpoll.internal.CondorcetPollResultsCalculator
+com.xwiki.xpoll.internal.MultiSelectionPollResultsCalculator
+com.xwiki.xpoll.internal.SingleSelectionPollResultsCalculator
```

Comment:
You're missing ``DefaultPollResultsCalculator``. Are you sure the build is passing?

Reply:
Yes. It doesn't throw any errors.